src/task.coffee

```diff
@@ -100,6 +100,11 @@ class Task
     @childProcess.removeAllListeners()
     @childProcess.on 'message', ({event, args}) =>
       @emit(event, args...) if @childProcess?
+    # Catch the errors that happened before task-bootstrap.
+    @childProcess.stdout.on 'data', (data) ->
+      console.log data.toString()
```

Comment:
This only prints when there is output before the `task-bootstrap` script starts to run, e.g. when there is regression in atom-shell. So as long as atom-shell works as expected, there will be nothing printed.